lib/rubocop/config_obsoletion.rb

```diff
@@ -8,7 +8,8 @@ class ConfigObsoletion
     COP_RULE_CLASSES = {
       'renamed' => RenamedCop,
       'removed' => RemovedCop,
-      'split' => SplitCop
+      'split' => SplitCop,
```

Comment:
I just realized that in the spirit of puns we should change "removed" to "retired". 😆 Probably we can also do something about "transfers between departments". 😆

Reply:
Hahaha yes!